res/skins/Deere (64 Samplers)/skin.xml

```diff
@@ -8,38 +8,48 @@
-      <attribute config_key="[Master],num_samplers">64</attribute>
+      <attribute config_key="[Master],num_samplers">16</attribute>
       <attribute config_key="[Master],num_preview_decks">1</attribute>
-      <attribute config_key="[Library],show_library">1</attribute>
-
-      <!--Skin settings-->
       <attribute persist="true" config_key="[Deere],show_parallel_waveforms">1</attribute>
-      <attribute persist="true" config_key="[Microphone],show_microphone">0</attribute>
-      <attribute persist="true" config_key="[PreviewDeck],show_previewdeck">0</attribute>
-
-      <attribute persist="true" config_key="[Master],show_4decks">0</attribute>
-      <attribute persist="true" config_key="[Master],show_4effectunits">0</attribute>
-      <attribute persist="true" config_key="[Master],show_coverart">1</attribute>
-      <attribute persist="true" config_key="[Master],show_spinnies">0</attribute>
-      <attribute persist="true" config_key="[Deere],show_starrating">0</attribute>
+      <!-- Decks -->
+      <attribute persist="true" config_key="[Skin],show_4decks">0</attribute>
+      <attribute persist="true" config_key="[Skin],show_8_hotcues">0</attribute>
+      <attribute persist="true" config_key="[Skin],show_coverart">1</attribute>
+      <attribute persist="true" config_key="[Skin],show_spinnies">1</attribute>
+      <attribute persist="true" config_key="[Skin],show_big_spinny_coverart">1</attribute>
+      <attribute persist="true" config_key="[Skin],show_rate_controls">1</attribute>
+      <attribute persist="true" config_key="[Skin],show_starrating">1</attribute>
       <attribute persist="true" config_key="[Deere],show_track_info">1</attribute>
       <attribute persist="true" config_key="[Deere],show_bpm_info">1</attribute>
-      <attribute persist="true" config_key="[Deere],show_more_hotcues">0</attribute>
-
+      <!-- Mixer -->
       <attribute persist="true" config_key="[Master],show_mixer">1</attribute>
-      <attribute persist="true" config_key="[Deere],show_crossfader">1</attribute>
-      <attribute persist="true" config_key="[Master],show_eqs">1</attribute>
-      <attribute persist="true" config_key="[Master],show_killswitches">0</attribute>
-      <attribute persist="true" config_key="[Master],show_faders">1</attribute>
-      <attribute persist="true" config_key="[VinylControl],show_vinylcontrol">0</attribute>
-
-      <attribute persist="true" config_key="[Samplers],show_samplers">1</attribute>
-      <attribute persist="true" config_key="[Samplers],show_expanded">0</attribute>
-
-      <attribute persist="true" config_key="[EffectRack1],show">1</attribute>
-      <attribute persist="true" config_key="[Master],show_superknobs">0</attribute>
+      <attribute persist="true" config_key="[Skin],show_eq_knobs">1</attribute>
+      <attribute persist="true" config_key="[Skin],show_eq_kill_buttons">1</attribute>
+      <attribute persist="true" config_key="[Skin],show_xfader">1</attribute>
```

Comment:
`[Skin],show_crossfader`? I don't care much either way.

Reply:
it was `[Master],show_xfader` in LateNight, that's where I started